mlocal/frags/Makefile.stub

```diff
@@ -170,7 +156,7 @@ test:
 	@echo "       PASS"
 	@echo " TEST go test"
 	$(V)export CGO_CPPFLAGS="$(cgo_CPPFLAGS)" CGO_LDFLAGS="$(cgo_LDFLAGS)" && \
-		cd $(SOURCEDIR) && go test -tags $(go_TAG) -cover -race -timeout 60s ./...
+		cd $(SOURCEDIR) && go test -count=1 -tags $(go_TAG) -cover -race -timeout 60s ./...
```

Comment:
I'm assuming the `-count=1` flag is to disable result caching? Do we really want to do that?

Reply:
Yes, to disable caching. Since I'm using `tests/dummy_test.go` as a testing framework for Singularity (rather than as a Go unit test) `go test` will not re-run the test on that package unless the code in that package changes. However, it doesn't realize that the result of that test is dependent on other code in the tree changing, so a cached test will give incorrect results. A way to solve this would be to stop using `go test` in this sort-of not intended way.

Reply:
OK, gotcha.

Reply:
Is `GOCACHE=off go test ...` a better way of making that explicit?

Reply:
From the help:

> The idiomatic way to disable test caching explicitly is to use -count=1.

Seems a bit weird to me, but who am I to argue with the `Go`ds?